core/src/main/java/io/grpc/internal/TransportSet.java

```diff
@@ -222,30 +232,32 @@ private boolean isAttachedToActiveTransport() {
     @Override
     public void transportReady() {
       synchronized (lock) {
-        log.info("Transport '" + transport + " is ready");
+        log.info("Transport '" + transport + "' for " + server + " is ready");
```

Comment:
These should really use the longer form: `log.log(Level.INFO, "Transport {0} for {1} is ready", transport, server);`

We should probably reduce these to debug? Or at least plan on doing so?